context.go

```diff
@@ -16,8 +16,6 @@ import (
 // Due to the limitations of passing pointers to C from Go we need to create
 // a registry so that we can lookup the Context from any given callback from V8.
 // This is similar to what is described here: https://github.com/golang/go/wiki/cgo#function-variables
-// To make sure we can still GC *Context we register the context only when we are
```

Comment:
Ah, this is why it didn't do this before, which doesn't apply now that there is no GC finalizer for Context.  I guess we might need to surround v8 calls in the future again if we decide to bring back Context GC